twig/BS3/fields/selectMultipleInput.twig

```diff
@@ -23,13 +23,13 @@
             {% endfor %}
         </select>
         <span class="input-group-btn">
-            <button class="btn btn-success tool_button" id="{{ id }}-all" type="button" title="Select All">
-                <i class="fa fa-check"></i>
+            <button class="btn btn-success tool_button" id="{{ id }}-all" type="button">
```

Comment:
We want the title on this so people can hover over and know what it does.  This control isn't just for screen readers.

Reply:
Got it!